docs/api/README.md

```diff
@@ -7,33 +7,16 @@ has_children: true
 # Delphi Epidata API
 
 This is the home of [Delphi](https://delphi.cmu.edu/)'s epidemiological data
-API for tracking epidemics such as influenza, dengue, and norovirus. Note that
-our work on COVID-19 is available in the [COVIDcast Epidata API documentation](covidcast.md).
+API. See the [COVIDcast Epidata API documentation](covidcast.md) for data
+specific to COVID-19.
 
 ## Contributing
 
-If you are interested in contributing:
-
-- For development of the API itself, see the
-  [development guide](../epidata_development.md).
-- To suggest changes, additions, or other ways to improve,
-  [open an issue](https://github.com/cmu-delphi/delphi-epidata/issues/new)
-  describing your idea.
+See https://github.com/cmu-delphi/delphi-epidata#contributing
```

Comment:
Most of the API docs pages direct users to `README.md#contributing`; if we don't want to send users on a goose chase this PR should change those links as well